ios/FluentUI/Vnext/AvatarGroup/AvatarGroup.swift

```diff
@@ -119,6 +119,7 @@ public struct AvatarGroup: View {
         let avatars: [MSFAvatarStateImpl] = state.avatars
         let maxDisplayedAvatars: Int = avatars.prefix(state.maxDisplayedAvatars).count
         let overflowCount: Int = (avatars.count > maxDisplayedAvatars ? avatars.count - maxDisplayedAvatars : 0) + state.overflowCount
+        let isLeftToRight = NSLocale.characterDirection(forLanguage: NSLocale.autoupdatingCurrent.languageCode!) == .leftToRight
```

Comment:
Forced unwrapping in Swift is (almost) always a hint that something unsafe is going on. The fact that this is optional suggests that there may be locales that have no language code, and thus this line will crash. Can we make this have a safer fallback?